f2/src/components/formik/datePicker/index.js

```diff
@@ -39,34 +39,31 @@ export const DatePicker = ({ field, form, ...props }) => {
         <Form.Text className="input-help-text">{props.helpText}</Form.Text>
       </Row>
 
-      <Row>
-        {(form.errors[field.name + 'Day'] ||
-          form.errors[field.name + 'Month'] ||
-          form.errors[field.name + 'Year']) && <Error>Invalid date</Error>}
-        {form.status.errors && form.status.errors[field.name + 'Error'] && (
-          <Error>{form.status.errors[field.name + 'Error']}</Error>
-        )}
-      </Row>
-
       <Row className="date-group">
         <Field
           name={field.name + 'Day'}
           component={DateEntry}
           id={props.id + 'Day'}
           value={props.day}
```

Comment:
Do we really need this line of code?
value={props.day}
It applies to month and year as well

Reply:
Tested this and it is not necessary. Update has been pushed.